src/main/resources/db/migration/V1_57__supplier_assessment.sql

```diff
@@ -0,0 +1,15 @@
+create table supplier_assessment(
+    id uuid not null,
+    referral_id uuid,
```

Comment:
can we add a unique key on referral id - meaning there can only ever be one supplier assessment per referral?